pymess/management/commands/send_messages_batch.py

```diff
@@ -64,13 +69,19 @@ def _print_result(self, title, message_pks):
         else:
             self.stdout.write('{}: {}'.format(title, len(message_pks)))
 
+    def _is_system_shutting_down(self):
+        return self.shutdown_detection_function and self.shutdown_detection_function()
```

Comment:
nitpick: I would return bool for all cases. This returns `True`/`False` (when: `self.shutdown_detection_function is not None`) or `None` (when: `self.shutdown_detection_function is None`).